examples/CMakeLists.txt

```diff
@@ -1,15 +1,10 @@
-
-#######################
-#######################
-#
-#  Define Executables
-#
-#######################
-#######################
-
-FIND_PACKAGE(ITK REQUIRED COMPONENTS PrincipalComponentsAnalysis)
-
-include_directories( ${PrincipalComponentsAnalysis_SOURCE_DIR}/Source )
+IF (NOT ITK_SOURCE_DIR)
```

Comment:
I think we only want (untested):

```
if(NOT ITK_SOURCE_DIR)
  find_package(ITK REQUIRED COMPONENTS PrincipalComponentsAnalysis)
  include(${ITK_USE_FILE}
else()
  include_directories( ${PrincipalComponentsAnalysis_SOURCE_DIR}/Source )
endif()
```

This is an example -- in should not call `itk_module_impl()`